ksql-engine/src/main/java/io/confluent/ksql/structured/SchemaKGroupedStream.java

```diff
@@ -184,14 +187,28 @@ private KTable aggregateWindowed(
     final KTable<Windowed<Struct>, GenericRow> aggKtable = ksqlWindowExpression.applyAggregate(
         kgroupedStream, initializer, aggregator, materialized);
 
+    //Apply mapper to get final result of aggregation, might be no-op
```

Comment:
instead of this, I'd explain why we need to run this _before_ the WindowSelectMapper, which is kinda special